### DISCUSSION TITLE
Animate GPX track

### DISCUSSION THREAD

**Discussion by vearutop** [StrandedKitty/streets-gl#137] [Q&A] [answered]
Is there an API to move the camera along predefined markers?

**Comment by StrandedKitty** [comment, marked as answer]
No, there's no API for this. You would have to implement a custom [ControlsNavigator](https://github.com/StrandedKitty/streets-gl/blob/165207de478bbda6593e48c9bbd94f5840ef2917/src/app/controls/ControlsNavigator.ts) fo this.
